src/EnergyPlus/MixedAir.cc

```diff
@@ -4328,6 +4347,16 @@ namespace MixedAir {
 
 		}
 
+		if ( OAController( OAControllerNum ).CoolCoilFreezeCheck ) {
+			MaxOAFracBySetPoint = min( max( MaxOAFracBySetPoint, 0.0 ), 1.0 );
+			OAController( OAControllerNum ).MaxOAFracBySetPoint = MaxOAFracBySetPoint;
```

Comment:
Merge is ok. Using your original test file with the maxOA limited to 0.2, there are values in Air System Outdoor Air Maximum Flow Fraction for some hours in April.  With my version of the test file without the 0.2 limit, these are all zero, but there are times where the OA fraction is different from the case without freezing protection. That's where I got confused.  And these happen at a time when the cooling coil isn't even running.
![image](https://cloud.githubusercontent.com/assets/4991075/13150919/615ee60a-d62d-11e5-83aa-70b6f2b4b106.png)